webharness/harness.js

```diff
@@ -76,24 +101,69 @@ function wait_for_message() {
 }
 
 /*
+ * Return a promise for the next available message
```

Comment:
Could you please update the comment here and on wait_for_test_messsage() so that they don't look like copy and paste any more? ;-)